src/styles/utils/_mixins.scss

```diff
@@ -1,6 +1,26 @@
-@mixin hover($_property, $_toValue) {
```

Comment:
Remove the deleted lines from 1 to 4, as they are not relevant to the current task and should not be included in the pull request.